convergence.md

```diff
@@ -1,3 +1,501 @@
 ## Convergence Process
 
-(placeholder)
+This document is intended to serve as the beginnings of a guide for merging
+the existing joyent/node and iojs/io.js repositories
+
+**This is a draft**. Nothing in this document should be considered binding or final.
+
+## Organization Overview
+
+### Joyent
+
+#### Organization
+
+The github.com/joyent organization belongs to Joyent, Inc and is used to support a broad number of Joyent's projects beyond Node.js.
+
+The organization currently consists of 53 members, not all of whom are Joyent employees.
+
+There are two github teams defined within the github.com/joyent organization: Owners and node-coreteam. The current members of the node-coreteam include:
+
+* http://github.com/tjfontaine
+* http://github.com/misterdjules
+* http://github.com/orangemocha
+* http://github.com/chrisdickinson
+* http://github.com/cjihrig
+* http://github.com/trevnorris
+* http://github.com/indutny
+* http://github.com/mhdawson
+* http://github.com/jasnell
+* http://github.com/srl295
+
+#### Repositories
+
+Node and several of it's dependencies / related projects currently reside within the http://github.com/joyent organization.
+
+* http://github.com/joyent/node
+* http://github.com/joyent/node-website
+* http://github.com/joyent/node-advisory-board
+* http://github.com/joyent/http-parser
+* http://github.com/joyent/node-tracing ?
+* http://github.com/joyent/docker-node ?
```

Comment:
What is the meaning of "?" here? If it's "should this belong to the Node.js foundation once it's created?", I would say yes for docker-node. node-tracing seems like an experiment in coming up with some tracing tools based on async listener, I'm not sure it should move over.